go.mod

```diff
@@ -170,3 +170,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/crossplane/crossplane-runtime => github.com/sttts/crossplane-runtime v0.0.0-20231006112446-53296455f75d
```

Comment:
We can't merge this with this `replace` statement in place. 😄 Did the changes you need in c/crossplane-runtime get merged?

Reply:
See https://github.com/crossplane/crossplane-runtime/pull/554. Not merged yet.

Reply:
FYI, just merged that.